python-backend/app/api/mines/variances/resources/variance_list.py

```diff
@@ -0,0 +1,99 @@
+from werkzeug.exceptions import BadRequest
```

Comment:
I see this differently, do we want the list and singular resources to be in the same file or separate files? I see this already done inconsistently. I'm on the fence but think i'd lean to having them on the same file cause they both function to serve one purpose.

Reply:
I get the documents being separate. but just variance and variance_list.

Reply:
Hey Jason, I actually split these because you had requested it in my original Variances PR and since I was splitting out documents, I decided to split everything. I guess your opinion has changed?